ee/server/service/teams.go

```diff
@@ -129,6 +137,13 @@ func (svc *Service) ModifyTeam(ctx context.Context, teamID uint, payload fleet.T
 		if *payload.Name == "" {
 			return nil, fleet.NewInvalidArgumentError("name", "may not be empty")
 		}
+		l := strings.ToLower(*payload.Name)
+		if l == strings.ToLower(fleet.ReservedNameAllTeams) {
+			return nil, fleet.NewInvalidArgumentError("name", "may not be all teams")
+		}
+		if l == strings.ToLower(fleet.ReservedNameNoTeam) {
+			return nil, fleet.NewInvalidArgumentError("name", "may not be no team")
```

Comment:
dumb question, why these have a different error message? the two below look better:

> "All teams" is a reserved team name`

Reply:
@roperzh  I think I was just trying to be consistent with the other errors in this part of the code. I can update it to be consistent.